test/common/event/dispatcher_impl_test.cc

```diff
@@ -535,7 +535,9 @@ TEST_F(DispatcherImplTest, IsThreadSafe) {
 
 class TestFatalAction : public Server::Configuration::FatalAction {
 public:
-  void run(const ScopeTrackedObject* /*current_object*/) override { ++times_ran_; }
+  void run(const std::vector<const ScopeTrackedObject*>& /*tracked_objects*/) override {
+    ++times_ran_;
```

Comment:
Could you add some tests that cover 0, 1, >1 entries in tracked_objects ?

Reply:
Done.